models/gantt.model.js

```diff
@@ -1,79 +1,79 @@
-import { Schema, model } from 'mongoose';
- 
+import { Schema, model } from "mongoose";
```

Comment:
Not changed anything in this file. Just formatted the code. Since this file is not used, I suggest to delete this.

Reply:
I will check